plugin/nulls.go

```diff
@@ -0,0 +1,78 @@
+package plugin
+
+import (
+	"github.com/apache/arrow/go/v13/arrow"
+	"github.com/apache/arrow/go/v13/arrow/array"
+	"github.com/apache/arrow/go/v13/arrow/memory"
+)
+
+// TODO: use in v4
```

Comment:
```suggestion
// TODO(v4): use in v4
```

I'm using this convention for easier grepping